Sources/Core/API/TextureManager.swift

```diff
@@ -7,6 +7,12 @@
 import Foundation
 import CoreGraphics
 
+public enum ErrorMode {
+  case IgnoreError
```

Comment:
Swift enum cases should have a leading lower case character, like `logError` 🙂

Reply:
Fixed 👍